src/taxman.py

```diff
@@ -475,6 +477,54 @@ def _evaluate_taxation_GERMANY(self, op: tr.Operation) -> None:
                 )
                 self.tax_report_entries.append(report_entry)
 
+        elif isinstance(op, tr.MarginFee):
+            # Fees for margin trading
+            self.remove_from_balance(op)
+            if in_tax_year(op):
+                taxation_type = "Kapitaleinkünfte aus Margin Trading"
+                report_entry = tr.MarginReportEntry(
+                    platform=op.platform,
+                    amount=op.change,
+                    coin=op.coin,
+                    utc_time=op.utc_time,
+                    interest_in_fiat=-self.price_data.get_cost(op),
+                    taxation_type=taxation_type,
+                    remark=op.remark,
```

Comment:
It might be smart to add a remark that this is a Margin Fee.

Reply:
Added remarks to all three operation types (fee, gain, loss).